### ISSUE TITLE
Is there any way to get nested folders in the sidebar?

### ISSUE THREAD

**Issue by danielrosehill** [imfing/hextra#495]
Is there any way to get nested levels of folders in the sidebar?

Like to show

```
- Folder
- - Subfolders  
--- Doc 1  
--- Doc 2  
```

Ie, you click into `folder` which undocks `subfolders` which then undock the individual files.

**Comment by imfing** [comment]
Thanks for reaching out!

I think you’re asking if you could expand child folders/files by clicking on the parent folders, without the parent folder itself being a page.

that is not supported currently. The sidebar entries are based on the Hugo file structure.

https://imfing.github.io/hextra/docs/guide/organize-files/#directory-structure

it might be possible once it supports reading sidebar structure from a different source than being auto generated.

**Comment by danielrosehill** [comment]
@imfing Yup, indeed. Pity, but thanks for getting back to me (and thanks for the link and docs, they're excellent!)
